package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "./node_modules/eslint/bin/eslint.js ./*.js ./assets ./deserializers ./serializers ./services ./test",
     "test": "./node_modules/mocha/bin/mocha 'test/**/*.js'",
+    "test:ci": "ENV=.env-test-min-version.ci npm run test && ENV=.env-test-max-version.ci npm run test",
```

Comment:
[Specs] There is something that I really don't like in this solution is that not all the test needs to have different environment. Why not just having multiple environment variables and in the test do something like a foreach (Or an helper to do it) Like that we will have the context in the `describe` and  we don't have to run twice the test for the tests that does not need it

Reply:
That' a good point indeed.

Reply:
We will think about it before continue 👍

Reply:
I've just produced a new version where tests are categorized and better selected 👍